ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -84,7 +126,7 @@ void RSkComponentTextInput::OnPaint(SkCanvas *canvas) {
   }
 
   drawShadow(canvas, frame, borderMetrics, textInputProps.backgroundColor, layer()->shadowOpacity, layer()->shadowFilter);
-  drawTextInput(canvas, component.layoutMetrics, paraBuilder, textInputProps);
+  drawTextInput(canvas, component.layoutMetrics, paraBuilder, textInputProps, cursor_);
```

Comment:
if drawTextInput is made class member , then we dont need to pass this cursor_ here